### ISSUE TITLE
Unknown Issue with Eiki Projector

### ISSUE THREAD

**Issue by jrobb006** [bitfocus/companion-module-pjlink#76]
Hello we have updated our streamdeck to version 3.1.1 after doing that we no longer are able to turn on our EIKI EK-621W projectors via pjlink. The connection is showing green, however none of the variables are reading. The ip addresses are correct. We then reverted back to the last stable 2.4 version and are having the same issues. Any help would be greatly appreciated. Thanks!

**Comment by coryarobbins** [comment]
I also have an Eiki projector (EK-720LU)  that I'm unable to power on or off.  The status reads correctly, and the variables appear to work, but on the latest beta and the latest stable version, I get this error when attempting to change the power state: `Projector reported %1POWR: Out of parameter`

It does work when I use the ProjectorManager app for mac.  On the latest stable version, I was able to get it to power on initially, but powering off hasn't worked at any point.  The most recent version doesn't seem to work in either regard.  I have tried it with and without a required password.
